android-sdk/src/test/java/org/sagebionetworks/bridge/android/manager/auth/InMemoryDAO.java

```diff
@@ -15,7 +18,7 @@
  * Created by jyliu on 1/20/2017.
  */
 @AnyThread
-class InMemoryDAO implements AuthenticationManager.DAO {
+class InMemoryDAO implements AccountDAO {
```

Comment:
Should this class be deleted?  Seems odd to have two separate classes implementing AccountDao